tests/test_notifications.py

```diff
@@ -174,3 +231,30 @@ def test_addon_install_complete(
     """Complete add-on installation and close notification."""
     complete_notification.close()
     browser.wait_for_notification(None)
+
+
+def test_failed_installation_notification(
+    failed_notification: AddOnInstallFailed,
+) -> None:
+    """Test that a failed installation notification is shown for a corrupt add-on."""
+    error_text = "The add-on downloaded from this site could not be installed because it appears to be corrupt."
+    assert failed_notification.error_message == error_text
+
+
+def test_close_failed_notification(
+    browser: BrowserWindow, failed_notification: AddOnInstallFailed
+) -> None:
+    """Close Failed Notification"""
+    failed_notification.close()
+    browser.wait_for_notification(None)
+
+
+@pytest.mark.parametrize(
+    "firefox_options", [{"page_load_strategy_none": True}], indirect=True
+)
+def test_progress_notification_downloading(
+    browser: BrowserWindow, progress_notification: AddOnProgress
+) -> None:
+    """Verify downloading status is reported correctly."""
+    description = progress_notification.is_downloading
+    assert description is not None
```

Comment:
`is_downloading` is said to be type `bool`. Is it hard to check if it's true or has been set to true?

Reply:
@b4handjr Oh, not at all. Let me change it to `assert description is True` to clarify it.